### ISSUE TITLE
Extension will fail to load without enough elements in independentPairColors setting.

### ISSUE THREAD

**Issue by Arithmeticae** [CoenraadS/BracketPair#8]
If the settings file is modified such that the orphan color is moved into the color group:
`   [
         "()",
         [
            "Red",
            "Gold",
            "Orchid",
            "LightSkyBlue"
         ]
   ],`
rather than:
`
   [
         "()",
         [
            "Red",
            "Gold",
            "Orchid",
            "LightSkyBlue"
         ],
      "Red"
      ],
`
the extension will fail to load as there aren't enough elements in that group.
`/C:/Program Files (x86)/Microsoft VS Code/resources/app/out/vs/workbench/workbench.main.js:121 [Extension Host] Activating extension `CoenraadS.bracket-pair-colorizer` failed: independentSetting [0] does not have 3 elements.`

Fundamentally this isn't an big issue, but it was difficult for me to determine what I was doing wrong without viewing the developer tools.

Great work so far though, it's coming together nicely.

**Comment by CoenraadS** [comment]
I agree, I need to figure out how JSON Schema works so I can assert its correctness in the settings windows, rather then relying on asserts in the code.
